app/controllers/users/users.controller.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../../models/index');
+const logger = require('../../logger');
+
+const { databaseError } = require('../../errors');
+const { encryptar } = require('./utils');
+
+exports.createUser = (req, res) => {
+  const { name, last_name, email, password } = req.body;
```

Comment:
in this case you could make use of the mappers https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#28--mappers